scipy/sparse/linalg/setup.py

```diff
@@ -11,7 +12,8 @@ def configuration(parent_package='', top_path=None):
     config.add_data_dir('tests')
 
     # PROPACK
-    config.add_subpackage('_propack')
+    if os.environ.get("USE_PROPACK"):
+        config.add_subpackage('_propack')
```

Comment:
This will break the opt-in for users. We can always build PROPACK, it should only be disabled at runtime. So please undo this change.